packages/eslint-plugin/src/configs/react.js

```diff
@@ -9,35 +8,25 @@ const config = {
   extends: [
     // Provides both rules and some parser options and other settings
     'airbnb',
+    // add typescript support for import plugin
+    'plugin:import/typescript',
     // Extended configs are applied in order, so these configs that turn other rules off should come last
     'prettier',
   ],
   parser: '@typescript-eslint/parser',
+  // parserOptions: {},
   plugins: [
     '@fluentui',
     '@rnx-kit',
     '@typescript-eslint',
     'deprecation',
-    'import',
     'jest',
     'jsdoc',
     'jsx-a11y',
     'react',
     'react-hooks',
   ],
   settings: {
-    // Some config suggestions copied from https://github.com/alexgorbatchev/eslint-import-resolver-typescript#configuration
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.tsx'],
-    },
-    'import/resolver': {
-      typescript: {
```

Comment:
this configuration was breaking `import/no-extraneous-dependencies` to work. now that `typescript` plugin is not being used at all and recommended configuration provided by `import` plugin is being used instead.